sky/backends/backend_utils.py

```diff
@@ -1826,6 +1835,20 @@ def run_ray_status_to_check_ray_cluster_healthy() -> bool:
         return record
 
     # All cases below are transitioning the cluster to non-UP states.
+
+    if len(node_statuses) == 0:
```

Comment:
style: https://google.github.io/styleguide/pyguide.html#2144-decision
```suggestion
    if not node_statuses:
```

Reply:
Wow I didn't know about this, I personally really dislike it but we should definitely be consistent.